src/products/components/ProductStocks/ProductStocks.tsx

```diff
@@ -131,6 +131,16 @@ const ProductStocks: React.FC<ProductStocksProps> = ({
     }
   };
 
+  const handleChange = e => {
+    onFormDataChange(e);
+    onFormDataChange({
+      target: {
+        name: "variantName",
+        value: e.target.value,
+      },
+    });
```

Comment:
Why do you call `onFormDataChange ` twice ?

Reply:
Hi @poulch, we are expecting variant name field changes when sku field is updated. we would like to keep this behavior.